utils/styleguide/github-hljs-theme.css

```diff
@@ -0,0 +1,90 @@
+.hljs {
```

Comment:
If you can't access variables, it would be good to comment this file with selected color names from https://garden.zendesk.com/css-components/utilities/color/.

Reply:
I've added the equivalent color values as comments. I also updated all of the colors to match an equivalent in the palette so it should be a little clearer now.